libfuse/fs_darwin.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libfuse
+
+import (
+	"bazil.org/fuse"
+	"bazil.org/fuse/fs"
+	"golang.org/x/net/context"
+)
+
+func (r *Root) platformLookup(ctx context.Context, req *fuse.LookupRequest, resp *fuse.LookupResponse) fs.Node {
```

Comment:
The error isn't really fatal and I don't want a missing external file to cause a real error.
But the platformLookup method should match the Lookup method signature.
And I'm changing those errors to Warning log messages, so it's clear they aren't fatal, and will add a comment.